README.md

```diff
@@ -27,14 +27,14 @@ Building REEF
 Requirements
 ------------
 
-  * Java 7 Development Kit
+  * Java 7 Development Kit or newer.
```

Comment:
Odd. It works for me routinely, and we have build servers using it. We might eventually have the opposite problem of introducing a Java8 dependency by accident :-)